app/views/admin/edition_legacy_associations/edit.html.erb

```diff
@@ -0,0 +1,28 @@
+<% page_title "Edit legacy associations: " + @edition.title %>
+<div class="row">
+  <h1><%= @edition.title %></h1>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <% initialise_script "GOVUK.adminEditionsForm", selector: '.js-edition-form', right_to_left_locales: Locale.right_to_left.collect(&:to_param) %>
+    <%= form_for @edition, url: admin_edition_legacy_associations_path(@edition), as: :edition, method: :put,
+        html: { class: edition_form_classes(@edition) } do |form| %>
+      <fieldset>
+        <legend>Associations</legend>
+
+        <div class="important-associations">
+          <% if @edition.can_be_related_to_policies? %>
+            <%= render 'admin/shared/policy_fields', form: form %>
+          <% end %>
+          <% if @edition.can_be_associated_with_topics? %>
+            <%= render 'topic_fields', form: form, edition: @edition %>
```

Comment:
Could we take the opportunity to rename this from "topic" to "policy_area", as that would make it a little clearer. The codebase is already inconsistent, so no need to rename everything.